client/human/HumanClientApp.cpp

```diff
@@ -908,7 +908,7 @@ void HumanClientApp::Reinitialize() {
 float HumanClientApp::GLVersion() const
 { return GetGLVersion(); }
 
-void HumanClientApp::StartTurn() {
+void HumanClientApp::StartTurn(const SaveGameUIData& ui_data) {
```

Comment:
is this parameter necessary, or can the relevant data be gotten within this function?

Reply:
Yes, `HumanClientApp::StartTurn` doesn't send TurnOrder itself, it calls `ClientApp::Start` where it required to have `SaveGameUIData`. But it also overrides `ClientApp::Start` so them have to have same signature.